sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRebaseDatetimeSuite.scala

```diff
@@ -374,6 +375,58 @@ abstract class ParquetRebaseDatetimeSuite
       }
     }
   }
+
+  test("SPARK-35427: datetime rebasing in the EXCEPTION mode") {
```

Comment:
do we need to clean up the existing tests to remove code that are being tested here?

Reply:
Let me find out the places where we check the default mode (`EXCEPTION`).

Reply:
I think we can avoid checking the default `EXCEPTION` mode at least in the places:
1. https://github.com/apache/spark/blob/0494dc90af48ce7da0625485a4dc6917a244d580/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRebaseDatetimeSuite.scala#L142-L146
2. https://github.com/apache/spark/blob/2bd32548f520ad3fc119e54ee3b4b4f27d33e046/external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala#L1835-L1837

@cloud-fan Let me know if you think that there are other places.